hadoop-ozone/ozone-manager/src/test/java/org/apache/hadoop/ozone/om/upgrade/TestOMUpgradeFinalizer.java

```diff
@@ -188,9 +198,11 @@ public void testFinalizationWithUpgradeAction() throws Exception {
     if (finalizer.isFinalizationDone()) {
       when(versionManager.getUpgradeState()).thenReturn(FINALIZATION_DONE);
     }
-    StatusAndMessages status = finalizer.reportStatus(CLIENT_ID, false);
+    StatusAndMessages status = finalizer.reportStatusAndMsgs(CLIENT_ID, false);
     assertEquals(FINALIZATION_DONE, status.status());
     assertFalse(status.msgs().isEmpty());
+    Status ret = finalizer.reportStatus();
+    assertEquals(status, ret);
```

Comment:
We need to assert equality of status.status(), and ret.
It might be more obvious if we rename the variable status to finalizerResponse or similar, but this is just an afterthought as I watch my code, to understand it from afar :) If you are up to do this rename, please rename the variable in other test methods as well.

Reply:
We might also assert equality of FINALIZATION_DONE, and status.status{} here.

Reply:
@fapifta thanks for the catch, I'll have better names for the variables in next push.